app/views/application/_browsing_index.html.erb

```diff
@@ -1,15 +1,17 @@
 <div class="container">
-<% @obituaries.each do |obit| %>
-  <div class="row" style= "background-color:white;">
-    <div class="col-md-3 p-4">
-      <%= link_to image_tag(obit.image_url), "/obituaries/#{obit.id}" %>
+<% cache(action: "index", action_suffix: "obituaries_index") do %>
```

Comment:
Just added this in all views, there are a few other ways to do it. The action suffix distinguishes between different caching on the page, the index is just the action

Reply:
![](https://media1.giphy.com/media/r62TBdD8aNIwU/giphy.gif?cid=5a38a5a2bl23ne0aysgcjwnfg5my6rcgwe70f0wwqzg8kuh8&rid=giphy.gif)